### ISSUE TITLE
phenotypic variance in slope breaks model

### ISSUE THREAD

**Issue by mdahirel** [mdahirel/pushed-pulled-2020-heritability-IBM#18]
the new version of the density-dispersal function (branch `alt-dispersal-travis09`) breaks down when phenotypic variation in slopes is allowed (`VP_slope` > 0), with an error message as follows:

```math operation produced a number too large for NetLogo
error while turtle 151 running EXP
  called by procedure SETUP-TURTLES
  called by procedure SETUP
  called by Button 'setup'
```

error happens at one of the steps where the script uses the dispersal function to estimate a dispersal rate, and is more likely the higher the VP.

I suspect it's because steep enough slopes lead, at high enough densities, to a number high enough in the denominator that exponentiating it breaks things. the problem only arise if VP is non-zero because our median slope is reasonable; it's just that by sampling we get a breaking value.

Solutions to consider: 
- see if changing density-dependency from N to N/K (and adjusting slope scale accordingly) changes things
- force slopes to be contained within bounds using logits
- cry?
